module/VuFind/src/VuFind/Captcha/Image.php

```diff
@@ -20,34 +20,38 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  *
  * @category VuFind
- * @package  View_Helpers
- * @author   Chris Hallberg <crhallberg@gmail.com>
+ * @package  CAPTCHA
+ * @author   Mario Trojan <mario.trojan@uni-tuebingen.de>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
- * @link     https://vufind.org/wiki/development Wiki
+ * @link     https://vufind.org Main Page
  */
-namespace VuFind\View\Helper\Bootstrap3;
+namespace VuFind\Captcha;
 
 /**
- * Recaptcha view helper
+ * Image CAPTCHA.
  *
  * @category VuFind
- * @package  View_Helpers
- * @author   Chris Hallberg <crhallberg@gmail.com>
+ * @package  CAPTCHA
+ * @author   Mario Trojan <mario.trojan@uni-tuebingen.de>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development Wiki
  */
-class Recaptcha extends \VuFind\View\Helper\Root\Recaptcha
+class Image extends LaminasBase
 {
     /**
-     * Constructor
+     * Generate HTML depending on CAPTCHA type.
      *
-     * @param \Laminas\Recaptcha\Recaptcha $rc     Custom formatted Recaptcha
-     * @param \VuFind\Config               $config Config object
+     * @return string
      */
-    public function __construct($rc, $config)
+    public function getHtml(): string
     {
-        $this->prefixHtml = '<div class="form-group">';
-        $this->suffixHtml = '</div>';
-        parent::__construct($rc, $config);
+        $id = $this->captcha->generate();
+        $imgUrl = '/vufind/cache/' . $id . $this->captcha->getSuffix();
```

Comment:
The hard-coded /vufind here is going to be a problem. You'll probably need to inject the base path through the constructor; I think one option would be to use `$container->get('ViewHelperManager')->get('url')->__invoke('home')`